packages/web3torrent/src/constants.ts

```diff
@@ -65,37 +65,24 @@ export const EmptyTorrent = ({
   wires: []
 } as unknown) as Torrent;
 
-// Mocked Constants
-export const mockTorrents: Array<Partial<Torrent>> = [
+// Pre Seeded Constants (by StateChannels team)
+export const preSeededTorrents: Array<Partial<Torrent>> = [
   {
-    name: 'Sintel',
-    length: 129302391,
-    numSeeds: 47,
-    numPeers: 12,
-    files: [],
+    name: 'Sintel.mp4',
+    length: 129241752,
+    infoHash: 'c53da4fa28aa2edc1faa91861cce38527414d874',
```

Comment:
Things that confuse me: 
1. The string that follows `btih` in the previous magnet link (which I assume means bit torrent info hash) is rejected by our tracker as an invalid infoHash, but the magnet link still works when you paste into our web3torrent client.
2. Changing the magnet link seems to change whether the paid streaming extension is used, or not.

Can you shed any light @spersico ? 


@spersico

Reply:
1 - What are you testing? And how is it getting rejected?.
According to [this wiki](https://en.wikipedia.org/wiki/Magnet_URI_scheme), the minimum viable magnet format is:
`magnet:?xt=urn:btih:INFOHASH`
The `btih` part is what clients and servers use to figure out what's the format of the hash. . Maybe you are not including that in the magnet that you send to the tracker?

2- Changing the magnet link how? We parse the magnet from the URL in [here](https://github.com/statechannels/monorepo/blob/25aa4b93cd877b897e20f112c8005278d7d6c2cd/packages/web3torrent/src/utils/magnet.ts#L5), and that's all that we do with that.

Reply:
1. I passed in that *part* of the magnetlink as an info hash in the bittorrent tracker client. In the logs of our tracker, I see an "Invalid infoHash" error (I never send it a magnet link). It doesn't matter so much, since I got it working by regenerating the magnet link and infoHash from our client. Perhaps I am just wrong to try and naively grab the infohash from the magnet link.

2. Using the magnet links that are in the mockConstants on `master`*, we seem to be able to connect to i) public trackers and ii) clients that do not run our paid streaming extension, and therefore do not demand any payment or channel setup. Probably ii is a consequence of i, and the fact the downloaders won't pay uploaders if they aren't prompted to do so! So I think I was wrong to say the paid streaming extension is not used; most likely it is only used by the downloader. Does this sound right to you?

3. The magnet link is different when we upload through our client (compared to the ones that are called `mockTorrents` on master). No big deal, but this tripped me up for a while. 

*(You can try this out on the deployed version of the app by clicking on one of the example torrents).

Reply:
1. I would, have to test what the tracker considers valid torrents. I don't quite know what it expects, currently.
2 is definitely correct. If the seeders don't seem to use the PSE extension, nor STOP and ask for funds, the leecher should run the download as is, without paying nor using anything extra from web3torrent. This was done on purpose, to maintain compatibility with non-web3torrent-torrents.
3 I think, was my first approach to make the url shorter and more readable, in a way. I don't quite remember right now what has been done exactly. 


I think that at the future, it would be a good idea to just drop the magnet format that we currently use, and just pass the infohash, name and size of the torrent as params or queryParams in the URL, instead of fragments in the URL.